### PR TITLE
[WIP] FUSETOOLS-2290 - make the camel version handling dynamically (decouple from runtime fully)

### PR DESCRIPTION
**Pull request by lhein**
this is the initial first draft. DO NOT MERGE! WIP!

### CODE REVIEW

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/jmx/camel/CamelFacadeSupport.java [hunk 1]
```diff
@@ -73,19 +71,18 @@ protected Object newProxyInstance(ObjectName objectName, Class<?> interfaceClass
...
-     * @throws IOException 
      */
-    protected Set<ObjectName> findCamelContexts(MBeanServerConnection connection, String managementName) throws MalformedObjectNameException, IOException {
+    protected Set<ObjectName> findCamelContexts(MBeanServerConnection connection, String managementName) throws Exception {
         String id = managementName != null ? managementName : camelContextManagementName;
 
         ObjectName name;
         if (id != null) {
-            name = new ObjectName(ORG_APACHE_CAMEL_CONTEXT + managementName + ",type=context,*");
+            name = new ObjectName("org.apache.camel:context=" + managementName + ",type=context,*");
```
Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/jmx/camel/CamelFacadeSupport.java]: it looks alike that you have reverted some commits, this one https://github.com/fusesource/fuseide/commit/61bbc27bd26779ce718be75458909ec3b2c8e5ba for this part

Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/jmx/camel/CamelFacadeSupport.java]: maybe its just not in sync with latest master? is that something you changed lately?

Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/jmx/camel/CamelFacadeSupport.java]: I checked the history and this branch is supposed to have only 2 commits behind the master.
I just found another missing commit which https://github.com/fusesource/fuseide/commit/ cbf63910654046eb3bceb652e42c1eaaa0c7b4b2

**Comment by lhein** [comment]
rebased with master

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
porting into 10.x branch. will reopen another PR

**Pull request closed without merging**